docs/en/testing.md

```diff
@@ -67,6 +67,26 @@ For example,
 ./tools/dist_test.sh configs/example_config.py work_dirs/example_exp/example_model_20200202.pth 1 --eval hmean-iou
 ```
 
+## Testing on Multiple Machines
+
+You can also launch a task on multiple machines connected to the same network.
+
+Say that you want to launch a job on two machines. On the first machine:
+
+```shell
+NNODES=2 NODE_RANK=0 PORT=${MASTER_PORT} MASTER_ADDR=${MASTER_ADDR} ./tools/dist_test.sh ${CONFIG_FILE} ${CHECKPOINT_FILE} ${GPU_NUM} [PY_ARGS]
```

Comment:
better to explain each parameter. such as
NNODES: number of machines, 
MASTER_ADDR: master  machine IP address,  
NODE_RANK:  the current machine index
GPU_NUM: the GPU num of current machie
GPU_NUM

Reply:
also better to link to `torch.distributed.launch`